CHANGELOG.md

```diff
@@ -9,12 +9,13 @@
 - [](https://github.com/vegaprotocol/vega/issues/xxxx) -
 
 ### 🛠 Improvements
-- [6795]https://github.com/vegaprotocol/vega/issues/6795 - max gas implementation
+- [6795](https://github.com/vegaprotocol/vega/issues/6795 - max gas implementation
 - [6641](https://github.com/vegaprotocol/vega/issues/6641) - network wide limits
 
 ### 🐛 Fixes
 - [6801](https://github.com/vegaprotocol/vega/issues/6801) - Fix internal data source validations
 - [6799](https://github.com/vegaprotocol/vega/issues/6799) - Move LP fees in transit to the network treasury
+- [6716](https://github.com/vegaprotocol/vega/issues/6716) - Use timestamp on all times fields
```

Comment:
Should this be in with breaking changes?

Reply:
Hum, maybe, will make it so.